pyplaato/models/keg.py

```diff
@@ -36,15 +36,15 @@ def __init__(self, attrs):
 
     def __repr__(self):
         return f"{self.__class__.__name__} -> " \
-               f"Bear Left: {self.beer_left}, " \
+               f"Beer Left: {self.beer_left}, " \
                f"Temp: {self.temperature}, " \
                f"Pouring: {self.pouring}"
 
     @property
     def date(self) -> float:
-        if self.__date is not None and not self.__date:
```

Comment:
This condition made it impossible to enter as if the value was `2022-10-01` it can't be None and also be an empty value.  This means it was never parsed and we always called the `super()` property which returns the current timestamp.